notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.set_flux_label('My Nobel Prize-winning spectrum', include_unit=False)\n",
+    "specviz.set_spec_label('My special name for the spectral axis', include_unit=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## i --  Write the current spectrum out to a new or existing image.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Query: new_image (for saving results of spectrum arithmetic)\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    " In response to  'i'  (write  current  spectrum)  this  parameter\n",
+    "        specifies  the  name  of a new image to create or existing image\n",
+    "        to overwrite.\n",
+    "\n",
+    "Modifications  to  the spectra being analyzed may be saved using the\n",
+    "    'i' key in a new, the current, or other  existing  spectra.   A  new\n",
+    "    image  is  created  as  a new copy of the current spectrum and so if\n",
+    "    the  current  spectrum  is  part  of  a  multiple   spectrum   image \n",
+    "    (including  a  long slit spectrum) the other spectra are copied.  If\n",
+    "    other spectra in the same image are then modified and saved use  the\n",
+    "    overwrite  option  to  replace then in the new output image.  If the\n",
+    "    output spectrum already exists then the \u000eoverwrite\u000f flag must  be  set\n",
+    "    to  allow  modifying  the  data.   This  includes  the case when the\n",
+    "    output spectrum is the same as the input  spectrum.   The  only  odd\n",
+    "    case  here  is  when  the  input spectrum is one dimensional and the\n",
+    "    output spectrum is two  dimensional.   In  this  case  the  user  is\n",
+    "    queried for the line to be written.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Query: overwrite\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Specviz Solution\n",
+    "\n",
+    "\n",
+    "The above three can be addressed by:\n",
+    "\n",
+    "1. In the notebook, `Spectrum1D.write` is the best way to write out a spectrum in a standard way.\n",
+    "2. In the desktop interface, a plugin to do `Spectrum1D.write` on a loaded spectrum is an option.  *However* it is not clear that this meets any needs if the arithmetic operations are not exposed in the UI (see the above discussion), so this should only be implemented if this is done and/or there is some other identified need."
+   ]
+  },
```

Comment:
It is important to either automatically write out results of data analysis plugins or have a button to do so at the bottom of each plugin.

Reply:
Good point.  This SPLOT function is bout saving the spectrum itself, but there's not really anywhere else we specifically raise the importance of this, so I added a note here.